games/ninswitch/game_imagerec.py

```diff
@@ -35,26 +34,24 @@
 
 
 class CaptureScreen(Switch):
-    def __init__(self, cap):
-        Path(SAVE_DIR_PATH).mkdir(parents=True, exist_ok=True)
-        self.cap = cap
+    def __init__(self):
```

Comment:
Add line `save_individual_frame = False` below `SAVE_ALL_FRAMES = False`, and remove the whole `def __init__` from this class.

Using lower case letters instead of `SAVE_INDIVIDUAL_FRAME` is more appropriate, because the value does not stay constant.